flake/nixosModules.nix

```diff
@@ -33,30 +33,27 @@
             The user to authenticate as with hydra.
           '';
         };
-
-        hydraPassFile = mkOption {
-          type = types.path;
-          default = "";
-          description = ''
-            A file containing the password to authenticate with against hydra.
-          '';
-        };
-
-        ghSecretFile = mkOption {
-          type = types.path;
```

Comment:
You are replacing `hydraPassFile` and `ghSecretsFile` with just `environmentFile` which breaks our usage in devx-ci, but it seems this was not intended as there is no commit that explicitly removes them. This PR is simply based on a much older master (c8d7023) from mid-2023 that didn't have 12f8d4c yet, which does the exact opposite. Is it possible to rebase on something newer? :smiley:

Reply:
This must have gotten lost in the iog -> zw3rk merge. I'll have a look next week (it's Friday night).

Reply:
@dermetfan I brought it back. Let me know if this is good enough.